src/components/SingleReview.jsx

```diff
@@ -27,6 +25,24 @@ const SingleReview = () => {
         })
     },[review_id]);
 
+    const addVoteReview = () => {
+      setSelectedReview((currReview) => {
+      currReview = {...currReview, votes: currReview.votes + 1};
+
+      return currReview;
+    });
+    patchVoteReview(review_id);
+  }
+
+  const minusVoteReview = () => {
+    setSelectedReview((currReview) => {
+    currReview = {...currReview, votes: currReview.votes - 1};
+
+    return currReview;
+  })
+  patchVoteReview(review_id);
```

Comment:
YOu might want to put a catch block on the end of the api call which will reset the vote in the event that the api promise rejects.

Reply:
added catch block, thanks